libr/core/cbin.c

```diff
@@ -1186,9 +1212,11 @@ static int bin_entry(RCore *r, int mode, ut64 laddr, int va, bool inifin) {
 			} else {
 				name = r_str_newf ("entry%i", i);
 			}
-			r_cons_printf ("\"f %s 1 0x%08"PFMT64x"\"\n", name, at);
-			r_cons_printf ("\"f %s_%s 1 0x%08"PFMT64x"\"\n", name, hpaddr_key, hpaddr);
-			r_cons_printf ("\"s %s\"\n", name);
+			char *n = __filterShell (name);
+			r_cons_printf ("\"f %s 1 0x%08"PFMT64x"\"\n", n, at);
```

Comment:
i think we can just solve this issue by just using ". the only char to be filtered then it's \n' and whitespace. i think we can simplify this approach a bit more